### PR TITLE
Bump ancient OkHTTP & Retrofit in Hooks (FINERACT-1252)

### PR DESCRIPTION
**Pull request by vorburger**
see FINERACT-1252

### CODE REVIEW

**Comment by vorburger** [comment]
@avikganguly01 @fynmanoj seeing your interest in Hooks functionality, would you like to code review this PR and merge if LGTY?

**Comment by vorburger** [comment]
@avikganguly01 @fynmanoj now the build passed.

**Comment by vorburger** [comment]
FTR: The build on this was green, but it needed a rebase following #1465... hopefully still green now.

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/hooks/service/HookWritePlatformServiceJpaRepositoryImpl.java
Comment (fynmanoj) [inline fineract-provider/src/main/java/org/apache/fineract/infrastructure/hooks/service/HookWritePlatformServiceJpaRepositoryImpl.java:287]: just wondering if this part is taken care ?  "  // Swallow error if it's because of method not supported or
                    // if url throws 404 - required for integration test,
                    // url generated on 1st POST request"

**Comment by fynmanoj** [comment]
LGTM

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/hooks/service/HookWritePlatformServiceJpaRepositoryImpl.java
Comment (vorburger) [reply fineract-provider/src/main/java/org/apache/fineract/infrastructure/hooks/service/HookWritePlatformServiceJpaRepositoryImpl.java:287]: I was also wondering, but the integration tests all still pass now, so looks like this actually wasn't needed.

**Comment by vorburger** [comment]
> LGTM 

@fynmanoj could you please reach out to @avikganguly01 to get him to review and merge this? To unblock you to rebase your PR. He can then review and merge your PR. (I'm trying to get the long list of Fineract committers to be more involved; there are too few active ones...)

File: integration-tests/src/test/java/org/apache/fineract/integrationtests/HookIntegrationTest.java
Comment (avikganguly01) [inline integration-tests/src/test/java/org/apache/fineract/integrationtests/HookIntegrationTest.java:100]: Weird host this Heroku URL is failing in Postman with trailing / but working in Retrofit with trailing /.

**Review by avikganguly01 — APPROVED** [review]

**Pull request merged**
